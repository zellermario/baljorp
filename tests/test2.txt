create SJ m1 0
create ISJ m2 0 2

addNeighbor m1 1 m2
addNeighbor m2 2 m1

create Eszkimo e m1

move e 1
pass e