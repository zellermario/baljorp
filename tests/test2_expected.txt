Fields:
SJ m1 (0) - object: x
	neighbours: [(1,m2)]
ISJ m2 (0,2) - object: x
	neighbours: [(2,m1)]

Buildings:

Actors:
> Eszkimo e (heat: 5) on m2
  Inventory: []